pallets/validators-manager/src/lib.rs

```diff
@@ -192,6 +190,7 @@ pub mod pallet {
 
     #[pallet::storage]
     #[pallet::getter(fn get_vote)]
+    #[pallet::unbounded]
```

Comment:
why do we need this?

Reply:
```
/// The optional attribute `#[pallet::unbounded]` declares the storage as unbounded. When
/// implementating the storage info (when `#[pallet::generate_storage_info]` is specified on
/// the pallet struct placeholder), the size of the storage will be declared as unbounded. This
/// can be useful for storage which can never go into PoV (Proof of Validity).
```

Reply:
I think applies in a Map, not a Vec. We are not making maps bounded.

Reply:
If all we have is a map then we shouldn't use this attribute because the idea behind it is to explicitly mention that a storage has `unbounded` items and is usually used with storage items that store vectors.

Reply:
The storage item is unbounded cause the map value items, contain unbounded vectors. This is addressed in the voting changes in #178 
The attribute is optional, and it was added to be informative.
Nonetheless, I can see that since it creates confusion, I will remove it since it is removed anyway on #178